ts/components/wallet/TransactionsList.tsx

```diff
@@ -99,11 +86,11 @@ class TransactionsList extends React.Component<Props> {
   };
 
   public render(): React.ReactNode {
-    if (this.props.isLoading) {
+    if (pot.isLoading(this.props.transactions)) {
```

Comment:
Are we going to add the loading error case in a different story?

Reply:
yes, that's for another PR :)